aws/resource_aws_vpc_peering_connection.go

```diff
@@ -254,10 +253,6 @@ func resourceAwsVPCPeeringUpdate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
-	if err := vpcPeeringConnectionWaitUntilAvailable(conn, d.Id(), d.Timeout(schema.TimeoutUpdate)); err != nil {
```

Comment:
no need to wait after modifying?

Reply:
No, according to the [AWS documentation](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-basics.html#vpc-peering-lifecycle), once `active` the state will only change to `deleting` or `deleted`.